docs/release-notes/mia-platform-v11.mdx

```diff
@@ -1,10 +1,145 @@
 ---
-id: mia-platform-v11-overview
+id: v11-overview
 title: Mia-Platform v11
 sidebar_label: v11 Overview
 image: "img/release-note-link-preview.png"
 ---
 
-import bannerImageUrl from './img/banner-overview-v10.png'
+import bannerImageUrl from './img/v11-banner.png'
 
-Welcome to Mia-Platform v11, a major full of new integrations and functionalities for an Enriched Control Plane ensuring a Frictionless Developer Experience.
+We are excited to announce the arrival of Mia-Platform v11, a major release that brings numerous new features and improvements to our key products: Mia-Platform Console, Fast Data, and Backoffice.
+This version is designed to provide more advanced and comprehensive solutions in the field of Internal Developer Platforms, Digital Integration Hubs, and Backoffice, with a specific focus on enhancing the Developer Experience.
+
+In the Console, we have introduced new sections for managing clusters, providers, machine2machine authentication, and webhooks, as well as support for GitHub Actions.
+These enhancements offer more comprehensive control and an even more frictionless development experience for users, improving the overall Developer Experience.
+
+In Mia-Platform Fast Data, the main innovations include the ability to store data in buckets, No Code configuration, and the introduction of new, ready-to-use connectors.
+These features make the product even more versatile and powerful, simplifying data management and integration.
+
+For Backoffice, we have enriched the product with new features, including the ability to configure it in No Code mode.
+These updates streamline daily management, allowing for greater control of pages and sections, and increasing operational efficiency.
+
+Don't miss the opportunity to explore all the incredible new features of Mia-Platform v11 and discover how we can help you transform your digital enterprise, providing an optimized development experience centered on developers' needs.
+Mia-Platform v11, Simplify Complexity.
+
+## Console
+
+### Providers Management
+
+We have developed Mia-Platform Console so that it can integrate properly with the best and most widely used tools and providers on the market. 
+Therefore, to better enhance this integration, we have introduced a proper governance system, and this feature has precisely this goal: to make it easier to keep constantly documented and ordered all the Providers you want the Console to use. Through the new dedicated section - named "Providers" - inside the "Company Overview" area it will be possible to manage the creation, configuration, and editing of the providers used by Mia-Platform Console (GitProvider, Secret Manager and CI/CD Tools).
```

Comment:
```suggestion
Therefore, to better enhance this integration, we have introduced a proper governance system, and this feature has precisely this goal: to make it easier to keep constantly documented and ordered all the Providers you want the Console to use. Through the new dedicated section - named "Providers" - inside the "Company Overview" area it will be possible to manage the creation, configuration, and editing of the providers used by Mia-Platform Console (Git Provider, Secret Manager and CI/CD Tools).
```

Reply:
should it be "provider**s**?

Reply:
no it is correct "provider" is the type of provider, the actual type might just be "Git" since the others are "Secret Manager" and not "Secret Manager Provider" however they are always referred to as "GIt Provider" so it's fine, I guess.